### ISSUE TITLE
Velp menu: menun otsikkopalkki liian leveä kun velp menu on minimized-tilassa

### ISSUE THREAD

**Issue by saviit** [TIM-JYU/TIM#3156]
Menun otsikkopalkki liian leveä kun velp menu on minimized-tilassa. 

Answerbrowserin userlistin ja Velp menun otsikkopalkit voisivat olla samankokoiset kun ne ovat minimized-tilassa.

**Comment by vesal** [comment]
> Menun otsikkopalkki liian leveä kun velp menu on minimized-tilassa.
>
> Answerbrowserin userlistin ja Velp menun otsikkopalkit voisivat olla samankokoiset kun ne ovat minimized-tilassa.
>

Taisin tehdä täsät toisen kortin, jossa ehdotin että entä jos minimoidut
"lentäisivätkin" jonnekin "kotiin" joka ei olisi ruudulla häiritsemässä, mutta
josta niitä helpsoti löytäisi.  Joku aika pieni "sticky" alavetolista jossakin
(missä?) nurkassa tms?

**Comment by saviit** [comment]
Joo, tein tästä tarkoituksella oman kortin ettei mene sekaisin tuon sun kortin kanssa. Sinänsä nämä kyllä liittyvät vähän samaan asiaan. 
Mulla on jotain ideaa tohon sun korttiin, mutten ole muilta hommilta vielä ehtinyt sinne kommentoida.
